third_party/libunwindstack/tests/PeCoffInterfaceTest.cpp

```diff
@@ -217,16 +237,77 @@ class PeCoffInterfaceTest : public ::testing::Test {
     return offset;
   }
 
+  uint64_t SetSectionHeaderAtOffset(uint64_t offset, std::string section_name) {
+    std::string name_in_header = section_name;
+    if (section_name.size() > 8) {
+      uint64_t previous_offset =
+          section_names_in_string_table_.empty() ? 0 : section_names_in_string_table_.back().first;
+      uint64_t previous_size = section_names_in_string_table_.empty()
+                                   ? 0
+                                   : section_names_in_string_table_.back().second.size() + 1;
+      uint64_t current_offset = previous_offset + previous_size;
+      name_in_header = std::string("/") + std::to_string(current_offset);
+      section_names_in_string_table_.emplace_back(std::make_pair(current_offset, section_name));
```

Comment:
Nit: Since you're using emplace_back, the make_pair is not needed.

Reply:
Done.